app/assets/stylesheets/projects.scss

```diff
@@ -0,0 +1,8 @@
+.visit-links p {
+  a {
+    color: #fff !important;
+    &:hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3